Chakra.tsx

```diff
@@ -0,0 +1,37 @@
+import {
+    ChakraProvider,
+    cookieStorageManager,
+    localStorageManager,
+  } from "@chakra-ui/react";
+  import { GetServerSidePropsContext } from "next";
+  import { ReactNode } from "react";
+  import theme from "./src/theme";
+  
+  interface ChakraProps {
+    cookies?: string;
+    children: ReactNode;
+  }
+  
+  export const Chakra = ({ children, cookies }: ChakraProps) => {
```

Comment:
Could you explain a bit what this component does? It's not in use, currently, as far as I can tell